engine/src/test/scala/cromwell/engine/workflow/WorkflowActorSpec.scala

```diff
@@ -142,10 +151,20 @@ class WorkflowActorSpec extends CromwellTestKitSpec with WorkflowDescriptorBuild
     "not run Finalization actor if aborted when in MaterializingWorkflowDescriptorState" in {
       val actor = createWorkflowActor(MaterializingWorkflowDescriptorState)
       deathwatch watch actor
+      copyWorkflowLogsPrope watch dummyLogCopyRouter
       actor ! AbortWorkflowCommand
       finalizationProbe.expectNoMsg(AwaitAlmostNothing)
       deathwatch.expectTerminated(actor)
     }
+
+    "copy workflow logs in the event of MaterializeWorkflowDescriptorFailureResponse" in {
+      val actor = createWorkflowActor(MaterializingWorkflowDescriptorState)
+      deathwatch watch actor
+      actor ! MaterializeWorkflowDescriptorFailureResponse(new Exception("Workflow Materialization Failed"))
+      finalizationProbe.expectNoMsg(AwaitAlmostNothing)
+      copyWorkflowLogsPrope.expectMsg(CopyWorkflowLogsActor.Copy(currentWorkflowId, mockDir))
+      deathwatch.expectTerminated(actor)
```

Comment:
It's intended that it shoots off the request before the WorkflowActor transitions to a FailedState...does that answer your question or are we talking about different things?